### DISCUSSION TITLE
Homepage not working

### DISCUSSION THREAD

**Discussion by OmariJuma** [vercel/next.js#68005] [Help]
### Summary

I have successfully deployed my project to vercel [link](https://biggest-orpin.vercel.app/) but when i try to visit the homepage i get this error message **Application error: a server-side exception has occurred (see the server logs for more information).

Digest: 1083418090**, but other routes like /signup, /login and /shop are working as expected. Kindly assist

### Additional information

```js
I have checked the runtime logs and its showing the request is 200 which means its ok. Someone kindly advice
```

**Comment by icyJoseph** [comment]
Your app is giving a 500 though, what is the homepage doing? Does it fetch data from an endpoint? Read a file?

**Reply by OmariJuma** [reply]
Yes its fetching data from the database, I have fixed it but I have bot updated vercel😀
